packages/foam-vscode/src/extension.ts

```diff
@@ -120,6 +125,73 @@ async function updateReferenceList() {
   }
 }
 
+async function openDailyNote() {
```

Comment:
General note on this file, the changes to extension.ts will majorly conflict with https://github.com/foambubble/foam/pull/83

@francishamel, if you could move the daily note functionality to its own file, this would be a lot easier to get merged in. As per our discussion on Discord, I can do this as well if needed.